code/js/controllers/SpotifyController.js

```diff
@@ -24,7 +24,7 @@
     var that = this;
     var selectorIndex;
 
-    if (window.location.hostname === "open.spotify.com") {
+    if (window.location.hostname === "open.spotify.com" || window.location.hostname === "play.spotify.com") {
```

Comment:
I always use `play.spotify.com` with the old player. I don't have the new player anymore, so I don't know when it should use the other `selectorIndex`.

My Player does have `#play-pause` as ID for for the Play/Pause Button:
```
<div id="controls">

		<div class="playback">
			<button id="previous" class="btn btn-link btn-icon spoticon-skip-back-24"></button>
	                <button id="play-pause" class="btn btn-play btn-icon"></button>
        	        <button id="next" class="btn btn-link btn-icon spoticon-skip-forward-24"></button>
		</div>
		<div class="extra">
			<button id="queue" title="Open Queue app"><span class="spoticon-queue-32"></span></button>
			<button id="shuffle" title="Shuffle"><span class="spoticon-shuffle-32"></span></button>
			<button id="repeat" title="Repeat"><span class="spoticon-repeat-32"></span></button>
			<span id="volume-show">
				<div id="volume" class="popover arrow-top arrow-position-right"><div id="volume-click"><div id="volume-bar"><div id="vol-bar-inner" style="width: 33px;"></div><span id="vol-position" style="left: 21.6px;"></span></div></div></div>
			</span>


		</div>
	</div>
```
Note that there is no more `.spoticon-play-32` or ` .spoticon-pause-32` as Classes.